readme.md

```diff
@@ -104,6 +109,14 @@ Default: `false`
 Flush the internal cache object.
 
 
+## Windows
+
+Though windows uses either `/` or `\` as its path separator, only `/`
```

Comment:
```
Though Windows may use `/`, `\`, or `\\` as path separators, you can **only** use forward-slashes (`/`) when specifying glob expressions. Any back-slashes (`\`) will be interpreted as escape characters instead of path separators.

This is common across many glob-based modules; see [`node-glob`](https://github.com/isaacs/node-glob#windows) for corroboration.
```